### ISSUE TITLE
Only load JS file when user is logged in

### ISSUE THREAD

**Issue by mundschenk-at** [wpmark/admin-bar-user-switching#11]
Currently the JS file is loaded on the frontend even when no user is logged in (and thus no admin bar displayed).

**Comment by wpmark** [comment]
Makes sense to me! Will look at implementing. Feel free to submit a pull request @mundschenk-at

**Comment by mundschenk-at** [comment]
@wpmark Done #12 :-)

**Comment by mundschenk-at** [comment]
@wpmark Thank you for partially fixing this. However, I think that #12 is superior to f0f19fd2bf06f72774d9281f4da3a210ed3ac2c9. Why do you prefer the other solution?

**Comment by mundschenk-at** [comment]
@wpmark Have you had any chance to look at this?
